src/CRM/CivixBundle/Resources/views/Code/mkdocs.yml.php

```diff
@@ -0,0 +1,21 @@
+site_name: <?php echo "$fullName\n"; ?>
+repo_url: https://lab.civicrm.org/extensions/FIXME
+theme: material
+
+pages:
+- Home: index.md
```

Comment:
@ejegg 

Would it make sense to point this at the README.md file? There's a lot of overlap between the `index.md` and `README.md` templates (wrt general overview and requirements).

https://github.com/totten/civix/blob/master/src/CRM/CivixBundle/Resources/views/Code/readme.md.php

There are a few differences wrt to install-instructions, usage, known issues, and future plans - however, I think these differences are more happenstance than essence.  Those topics all seem equally plausible as "quick intro material".

Reply:
It might make sense! At this point, I'm just trying to be consistent with the instructions at https://docs.civicrm.org/dev/en/latest/documentation/extensions/ .
In the README section on that page, it suggests that you might want to just make a symlink from README.md to docs/index.md. So I guess I'd want to leave this yml file as-is in any case, just start with a symlink.

Reply:
Without a file called "index.md" you won't get an autoloaded page when hitting the repo. There needs to be at least one file called "index" at least at the moment.

Reply:
Thanks @MikeyMJCO! over in PR #162 (the version that should be mergeable before the docs infrastructure upgrade) I left index.md in place and just added the explanatory comment suggested below.